app/views/SettingsView/index.js

```diff
@@ -1,183 +1,177 @@
+		flex: 1,
+		fontSize: 14,
+		marginStart: 20,
+		...sharedStyles.textColorNormal,
+		...sharedStyles.textRegular
+	},
+	sectionItem: {
+		backgroundColor: COLOR_WHITE,
+		paddingVertical: 16,
+		flexDirection: 'row',
+		alignItems: 'center'
+	},
+	sectionItemDisabled: {
+		opacity: 0.3
+	},
+	separator: {
+		height: StyleSheet.hairlineWidth,
+		backgroundColor: COLOR_SEPARATOR
+	},
+	sectionSeparatorBorder: {
+		borderColor: COLOR_BORDER,
+		borderTopWidth: 10
+	},
+	iconStyle: {
+		transform: [{ rotate: I18nManager.isRTL ? '90deg' : '270deg' }],
+		alignSelf: 'baseline',
+		marginEnd: 20
 	}
+});
 
-	constructor(props) {
-		super('SettingsView', props);
-		this.state = {
-			placeholder: {},
-			language: props.userLanguage ? props.userLanguage : 'en',
-			languages: [{
-				label: 'English',
-				value: 'en'
-			}, {
-				label: 'Português (BR)',
-				value: 'pt-BR'
-			}, {
-				label: 'Russian',
-				value: 'ru'
-			}, {
-				label: '简体中文',
-				value: 'zh-CN'
-			}, {
-				label: 'Français',
-				value: 'fr'
-			}, {
-				label: 'Deutsch',
-				value: 'de'
-			}, {
-				label: 'Português (PT)',
-				value: 'pt-PT'
-			}],
-			saving: false
-		};
-	}
+const renderSeparator = () => <View style={styles.separator} />;
 
-	shouldComponentUpdate(nextProps, nextState) {
-		const { language, saving } = this.state;
-		const { userLanguage } = this.props;
-		if (nextState.language !== language) {
-			return true;
-		}
-		if (nextState.saving !== saving) {
-			return true;
-		}
-		if (nextProps.userLanguage !== userLanguage) {
-			return true;
-		}
-		return false;
-	}
+export default class SettingsView extends Component {
+    static navigationOptions = ({ navigation }) => ({
+     headerLeft: <DrawerButton navigation={navigation} />,
+     title: I18n.t('Settings')
+    })
 
-	getLabel = (language) => {
-		const { languages } = this.state;
-		const l = languages.find(i => i.value === language);
-		if (l && l.label) {
-			return l.label;
-		}
-		return null;
+	static propTypes = {
+		navigation: PropTypes.object
 	}
 
-	formIsChanged = () => {
-		const { userLanguage } = this.props;
-		const { language } = this.state;
-		return !(userLanguage === language);
+	sections() {
+		const settings = [
+			{
+				data: [
+					{
+						withScreen: true,
+						title: 'Contact us',
+						screen: 'comming Soon',
+						isDeveloped: false
+					}, {
+						withScreen: true,
+						title: 'Language',
+						screen: 'LanguageView',
+						isDeveloped: true
+					}, {
+						withScreen: true,
+						title: 'Theme',
+						screen: 'comming Soon',
+						isDeveloped: false
+					}, {
+						withScreen: true,
+						title: 'Share this app',
+						screen: 'comming Soon',
+						isDeveloped: false
+					}
+				],
+				renderItem: this.renderNromalSettingItem
+			}, {
+				data: [{
+					withScreen: true,
+					title: 'License',
+					screen: 'comming Soon',
+					isDeveloped: false
+				}, {
+					withScreen: false,
+					title: 'Version: 3.4.1 (250)',
```

Comment:
add logic here same as implemented in the bottom of sidebar and show vision no. also remove the version no. from bottom of side nav

Reply:
done